.github/workflows/ci.yml

```
@@ -35,7 +35,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, windows-latest, macos-latest]
-        python-version: [3.6, 3.7, 3.8, 3.9]
+        python-version: [3.6, 3.7, 3.8, 3.9, 3.10]
```

```suggestion
        python-version: ["3.6", "3.7", "3.8", "3.9", "3.10"]
```

Versions are strings, not floating point numbers.